pkgs/beautyline-icons/default.nix

```diff
@@ -0,0 +1,59 @@
+{ breeze-icons
+, fetchFromGitLab
+, gnome-icon-theme
+, gtk3
+, hicolor-icon-theme
+, jdupes
+, lib
+, mint-x-icons
+, pantheon
+, stdenvNoCC
+,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "BeautyLine";
+  version = "unstable-2023-04-02";
+
+  src = fetchFromGitLab {
+    owner = "garuda-linux/themes-and-settings/artwork";
+    repo = "beautyline";
+    rev = "24052efcb887a58721ffef731181af65ee81b3c1";
+    sha256 = "sha256-6Nt7m/P0WUjoOetWLrh6pgkyg7FSLg1hGURgKgy6zdc=";
+  };
+
+  sourceRoot = "${src.name}";
+
+  nativeBuildInputs = [ jdupes gtk3 ];
+
+  # ubuntu-mono is also required but missing in ubuntu-themes (please add it if it is packaged at some point)
+  propagatedBuildInputs = [
+    breeze-icons
+    gnome-icon-theme
+    hicolor-icon-theme
+    mint-x-icons
+    pantheon.elementary-icon-theme
+  ];
+
+  dontDropIconThemeCache = true;
+
+  dontPatchELF = true;
```

Comment:
Are there ELF files in this derivation's `$out`?  :eyes:

Reply:
I used the existing package as base and kept things the same mostly.